services/src/main/java/org/exoplatform/officeonline/WOPIService.java

```diff
@@ -1064,6 +1084,12 @@ public String getMimeTypeByExtension(String extension) throws FileExtensionNotFo
         return entry.getKey();
       }
     }
+    if (extension.equals("wopitest")) {
```

Comment:
Not sure it's correct assumption. Let's use a default one `application/octet-stream` if it's not critical for Office services.

Reply:
Fixed: https://github.com/exo-addons/office-online/pull/13/commits/e93ddc3e549cd35e3bb596f7c5c6aeb0fdc73878